src/core/ddsc/include/dds/ddsc/dds_public_status.h

```diff
@@ -364,7 +364,7 @@ dds_get_offered_incompatible_qos_status (
  */
 DDS_EXPORT dds_return_t
 dds_get_subscription_matched_status (
-  dds_entity_t reader,
```

Comment:
This really only takes a reader, so I think "reader" is the better choice.

Reply:
Applied and fixed.